src/Components/Confirmation/ConfirmationSteps.tsx

```diff
@@ -127,7 +127,13 @@ function Expenses() {
 
   const allExpenses = () => {
     if (formData.expenses.length === 0) {
-      return <FormattedMessage id="confirmation.none" defaultMessage="None" />;
+      return (
+        <ConfirmationItem
+          label=""
```

Comment:
@IrynaKolh can you explain these ConfirmationItem with label=""? It seems based on this code `label &&` that the value won't display if label is an empty string? 
![image](https://github.com/user-attachments/assets/ed4eff9f-7c81-4b1b-b929-a4119038a7ab)